### ISSUE TITLE
not only a 4-digit number to secure the access of the app

### ISSUE THREAD

**Issue by UserXH726** [daper/nextcloud-passwords-app#43]
Hello,

I wish that I can choose a longer password and not only a 4-digit number to secure the app.

**Comment by GreyMarvel** [comment]
Agreed a stronger Master password is required, that uses letters, numbers and symbols would be great  addition.

**Comment by lbdroid** [comment]
This should really be delegated to the system authentication like the Nextcloud (files) application. I.e., if you have the system set to use pattern, then upon launching Nextcloud files, you are presented with a pattern request for the same pattern. The system handles the authentication. This actually makes the authentication easier to implement and a lot more flexible.

**Comment by Schengawegga** [comment]
I also think, that only 4 digits are not enough to keep the security on a level to use the app in business purposes.
This passcode strenght is very low and not secure enough for some reasons.
I know, that i should take care of a strong password at the lock screen of my phone, and i do that.
But i think, more security is even better than less security.
Could you please make a stronger passcode possible?
For example up to 20 digits with numbers, letters and a limited range of special characters, like semikolon and exclamation mark.
